app/uk/gov/hmrc/epayeapi/connectors/EpayeConnector.scala

```diff
@@ -39,21 +41,56 @@ case class EpayeConnector @Inject() (
   def getTotals(empRef: EmpRef, headers: HeaderCarrier): Future[ApiResponse[AggregatedTotals]] = {
     val url =
       s"${config.baseUrl}" +
-      s"/epaye" +
-      s"/${empRef.encodedValue}" +
-      s"/api/v1/totals"
+        s"/epaye" +
+        s"/${empRef.encodedValue}" +
+        s"/api/v1/totals"
 
     get[AggregatedTotals](url, headers)
   }
 
   def getTotalsByType(empRef: EmpRef, headers: HeaderCarrier): Future[ApiResponse[AggregatedTotalsByType]] = {
     val url =
       s"${config.baseUrl}" +
-      s"/epaye" +
-      s"/${empRef.encodedValue}" +
-      s"/api/v1/totals/by-type"
+        s"/epaye" +
+        s"/${empRef.encodedValue}" +
+        s"/api/v1/totals/by-type"
 
     get[AggregatedTotalsByType](url, headers)
   }
+
+  def getAnnualSummary(empRef: EmpRef, headers: HeaderCarrier, taxYear: Option[String]): Future[ApiResponse[AnnualSummaryResponse]] = {
+    val url =
+      s"${config.baseUrl}" +
+        s"/epaye" +
+        s"/${empRef.encodedValue}" +
+        s"/api/v1/annual-statement" + extractTaxYear(taxYear)
+        .map(TaxYear.asString)
+        .map(q => s"/$q")
+        .getOrElse("")
+
+    get[AnnualSummaryResponse](url, headers)
+  }
 }
 
+object EpayeConnector {
+  def extractTaxYear(taxYear: Option[String]): Option[TaxYear] = {
+    taxYear.flatMap {
+      case TaxYearPattern(year) => Some(year)
+      case _ => None
+    }
+  }
+
+  object TaxYearPattern {
+    lazy val pattern = """20(\d\d)-(\d\d)""".r
+    def unapply(taxYear: String): Option[TaxYear] = {
+      taxYear match {
```

Comment:
I would personally get rid of the unapply and put its body in the extractTaxYear directly